CMakeLists.txt

```diff
@@ -304,6 +307,10 @@ if(ENABLE_DBUSGATEWAY)
     install(PROGRAMS ${CMAKE_BINARY_DIR}/dbus-proxy/bin/dbus-proxy DESTINATION bin)
 endif()
 
+if(ENABLE_UNCRUSTIFY)
+    ## TODO: change this to use a config source
```

Comment:
This was left intentionally since I didn't start using this flag. I could remove the change completely since it doesn't do anything at the moment.